safe_transaction_service/history/services/collectibles_service.py

```diff
@@ -312,19 +312,19 @@ def _get_collectibles(
         if not addresses_with_token_ids:
             return [], 0
 
+        count = len(addresses_with_token_ids)
+        # TODO Paginate on DB
+        if limit is not None:
```

Comment:
Isn't this on db level? Django orm maps this to a sql query, right.

Reply:
I'm doing some magic to get all the collectibles for a Safe from DB: https://github.com/safe-global/safe-transaction-service/blob/master/safe_transaction_service/history/models.py#L516

Even for the Safes with most collectibles query is pretty fast so we decided not to write a new `COUNT(*)` query or even modify this one for using `LIMIT` and `OFFSET` for now. It's it not a difficult change and we should do it at some point